asking-good-questions/asking-good-questions.md

```diff
@@ -19,7 +19,9 @@ About 2 hours
 - Write (Check for Understanding)
 
 ### Motivation
-The ability to ask questions that lead to useful answers is a must-have skill for software engineers because they are constantly solving problems. Asking the right questions helps engineers arrive at solutions more quickly and make faster progress.
+The ability to ask questions that lead to useful answers is a must-have skill for software engineers because they are constantly solving problems. Asking the right questions helps engineers arrive at solutions more quickly and make faster progress. 
+
```

Comment:
questions - about

Reply:
question regarding silliest of doubts they have. or anything regarding company.
Do I need to add this ?

Reply:
Yes.  
questions—about should be questions -- about. it is not the same word.